.github/workflows/build-and-push-image.yml

```diff
@@ -109,28 +112,61 @@ jobs:
               core.setFailed(error.message);
             }
 
+  acr-import:
+    name: Import images to ${{ needs.set-env.outputs.environment }} ACR
+    needs: [ build-and-push-image, set-env ]
+    runs-on: ubuntu-22.04
+    environment: ${{ needs.set-env.outputs.environment }}
+    steps:
+      - name: Azure login with ACR credentials
+        uses: azure/login@v2
+        with:
+          creds: ${{ secrets.ACR_CREDENTIALS }}
+
+      - name: Run ACR Import
+        uses: azure/cli@v2
+        with:
+          azcliversion: 2.40.0
+          inlineScript: |
+            TAGS=(
+              ${{ needs.set-env.outputs.branch }}
+              ${{ needs.set-env.outputs.release }}
+              sha-${{ needs.set-env.outputs.checked-out-sha }}
+              latest
+            )
+            az config set extension.use_dynamic_install=yes_without_prompt
+            for tag in "${TAGS[@]}"
+            do
+              az acr import \
+                --name ${{ secrets.ACR_NAME }} \
+                --source "ghcr.io/${{ needs.set-env.outputs.github_repository_lc }}:$tag" \
+                --image "${{ env.DOCKER_IMAGE }}:$tag" \
+                --username ${{ github.actor }} \
+                --password ${{ secrets.GITHUB_TOKEN }} \
+                --force
+            done
+
   deploy-image:
     name: Deploy to ${{ needs.set-env.outputs.environment }} (${{ needs.set-env.outputs.release }})
-    needs: [ build-and-push-image, set-env ]
+    needs: [ build-and-push-image, set-env, acr-import ]
     runs-on: ubuntu-22.04
     environment: ${{ needs.set-env.outputs.environment }}
     steps:
       - name: Azure login with ACA credentials
         uses: azure/login@v2
         with:
-          creds: ${{ secrets.AZURE_ACA_CREDENTIALS }}
```

Comment:
Have you renamed all of the github secrets?

Reply:
Ah - good catch
For now I'll update the PR to use the `AZURE_` prefix for the existing `ACA` secrets, but we should probably update the secret names so that they are consistent with the other DfE/RSD repos